src/library/scala/ref/ReferenceWrapper.scala

```diff
@@ -20,10 +21,10 @@ trait ReferenceWrapper[+T <: AnyRef] extends Reference[T] with Proxy {
     if (ret eq null) throw new NoSuchElementException
     ret
   }
-  def clear() = underlying.clear()
-  def enqueue() = underlying.enqueue()
-  def isEnqueued = underlying.isEnqueued
-  def self = underlying
+  def clear(): Unit = underlying.clear()
+  def enqueue(): Boolean = underlying.enqueue()
+  def isEnqueued: Boolean = underlying.isEnqueued
+  def self: java.lang.ref.Reference[_ <: T] = underlying
```

Comment:
```suggestion
  def self: JaRef[_ <: T] = underlying
```

Reply:
Thanks - I went the other way, as `java.lang.ref.Reference` is used elsewhere, and removed the `JaRef` import